src/Client/Web/Bit.Client.Web.BlazorUI/Components/Inputs/BitDropDown.razor

```diff
@@ -55,21 +55,36 @@
              tabindex="0"
              role="listbox"
              aria-labelledby="@DropdownLabelId">
-            @foreach (BitDropDownItem item in Items)
+            @foreach ((BitDropDownItem item, Int32 i) in Items.Select((item, i) => (item, i)))
             {
                 if (item.ItemType == BitDropDownItemType.Header)
                 {
-                    <div class="@RootElementClass-head-@VisualClassRegistrar()">
+                    <div class="@RootElementClass-head-@VisualClassRegistrar()"
+                         id="@($"{DropDownId}{item.Text?.ToLower()}Header")">
                         @item.Text
                     </div>
                 }
                 else if (item.ItemType == BitDropDownItemType.Normal)
                 {
                     @if (IsMultiSelect)
                     {
-                        <BitCheckbox IsChecked="item.IsSelected"
+                        <BitCheckbox Class="@(Items.Any(prop=>prop.IsSelected && prop.Value.Equals(item.Value))
```

Comment:
please use `item.IsSeleted` instead of `Items.Any(prop=>prop.IsSelected && prop.Value.Equals(item.Value)`